perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py

```diff
@@ -840,9 +863,26 @@ def Prepare(benchmark_spec):
   # Setup common test tools required on the client VM
   vms[0].Install('sysbench05plus')
 
+
+  vms[0].mysql_svc_oltp_tables_count = FLAGS.mysql_svc_oltp_tables_count
+  vms[0].mysql_svc_oltp_table_size = FLAGS.mysql_svc_oltp_table_size
+
   # Prepare service specific states (create DB instance, configure it, etc)
   MYSQL_SERVICE_BENCHMARK_DICTIONARY[FLAGS.cloud].Prepare(vms[0])
 
+  metadata = {
+      'mysql_svc_oltp_tables_count': vms[0].mysql_svc_oltp_tables_count,
+      'mysql_svc_oltp_table_size': vms[0].mysql_svc_oltp_table_size,
+      'mysql_svc_db_instance_cores': FLAGS.mysql_svc_db_instance_cores,
+      'sysbench_warm_up_seconds': FLAGS.sysbench_warmup_seconds,
+      'sysbench_run_seconds': FLAGS.sysbench_run_seconds,
+      'sysbench_thread_count': FLAGS.sysbench_thread_count,
+      'sysbench_latency_percentile': FLAGS.sysbench_latency_percentile,
+      'sysbench_report_interval': FLAGS.sysbench_report_interval
+  }
+  DATA_LOAD_RESULTS.append(_PrepareSysbench(vms[0], metadata))
+  logging.info('Data Load Results: \n%s', DATA_LOAD_RESULTS)
```

Comment:
change this to 

print DATA_LOAD_RESULTS

which will output to stdout, which is much easier to read and consume. (logging.info will log to stderr which will mix this up with all other output, very noisy...)